portals/publisher/source/src/app/data/api.js

```diff
@@ -2820,10 +2820,11 @@ class API extends Resource {
      */
     static getCommonOperationPolicies() {
         const restApiClient = new APIClientFactory().getAPIClient(Utils.getCurrentEnvironment(), Utils.CONST.API_CLIENT).client;
-        const limit = Configurations.app.operationPolicyCount;
         return restApiClient.then(client => {
             return client.apis['Operation Policies'].getAllCommonOperationPolicies(
-                {limit},
+                {
+                    limit: 50,
```

Comment:
Shouldn't this come from `operationPolicyCount` config ?

Reply:
BTW what if there are more than 50 operation policies? 
Does the REST API support pagination?

Reply:
as discussed offline
![image](https://user-images.githubusercontent.com/3313885/158115169-b981d630-2242-4867-9366-f01f4a513ef4.png)

Reply:
We shall address this as a feature enhancement later on. Hence, marking it as resolved for now. Previously harcoded limit value is now read from the config.